xml/System.Windows.Forms/TaskDialogButton.xml

```diff
@@ -14,8 +14,28 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Represents a button control of a task dialog.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+ A button can either be a standard button (whose text is provided by the OS), or a custom button (or command link) where you can provide your own text.
+            
```

Comment:
I notice too that there are a lot of empty whitespace lines, with like 10 whitespace characters. These would fail our docs linters.